examples/request_api.py

```diff
@@ -53,23 +47,24 @@ def run_proc2():
 
 
 async def proc2_worker():
-    endpoint = Endpoint()
-    await endpoint.start_serving(ConnectionConfig.from_name('e2'))
-    await endpoint.connect_to_endpoints(
-        ConnectionConfig.from_name('e1'),
-    )
-    for i in range(3):
-        print("Requesting")
-        result = await endpoint.request(GetSomethingRequest())
-        print(f"Got answer: {result.payload}")
+    config = ConnectionConfig.from_name("e2")
+    async with AsyncioEndpoint.serve(config) as endpoint:
+        await endpoint.connect_to_endpoints(ConnectionConfig.from_name("e1"))
+        await endpoint.wait_until_any_remote_subscribed_to(GetSomethingRequest)
+
+        for i in range(3):
+            print("Requesting")
+            result = await endpoint.request(GetSomethingRequest())
+            print(f"Got answer: {result.payload}")
+
 
 if __name__ == "__main__":
 
-    multiprocessing.set_start_method('spawn')
+    multiprocessing.set_start_method("spawn")
```

Comment:
It might be worth commenting *why* on this assuming it's necessary for things to work properly.